### DISCUSSION TITLE
Cygwin support?

### DISCUSSION THREAD

**Discussion by edricgavinb** [jdx/mise#518] [Q&A]
I don't know enough about coding so sorry if it's already answered here: https://github.com/jdxcode/rtx/discussions/66,
but is there CYGWIN SUPPORT?

I tried...
curl https://rtx.pub/rtx-latest-linux-x64 > ~/bin/rtx
but ultimately got this error...
-bash: /home/m/bin/rtx: cannot execute binary file: Exec format error

**Comment by koppor** [comment]
Follow-up at https://github.com/jdx/mise/discussions/4113 and PR https://github.com/jdx/mise/pull/4048

**Comment by roele** [comment]
POSIX compatibility layers such as Cygwin or Git Bash do execute `exe` files. So you can install `mise` for Windows either [manually](https://mise.jdx.dev/installing-mise.html#windows-manual) or via package manager (scoop, winget).

There is no complete support for Cygwin or Git Bash, so activation (via shell script) will not work but shims should work and are good enough for most use-cases.
